Makefile.js

```diff
@@ -274,10 +286,14 @@ function publishSite() {
 /**
  * Updates the changelog, bumps the version number in package.json, creates a local git commit and tag,
  * and generates the site in an adjacent `website` folder.
+ * @param {Object} options Release options.
+ * @param {string} options.packageTag Tag that should be added to the package submitted to the npm registry.
  * @returns {void}
  */
-function generateRelease() {
-    ReleaseOps.generateRelease();
+function generateRelease({ packageTag }) {
```

Comment:
Sanity check: Is this function signature intentionally different than the one in #18871?

Reply:
Yes, we had (and still have on this branch) separate functions `function generateRelease` and `function generatePrerelease` which were later in the `main` branch merged into one (https://github.com/eslint/eslint/commit/c2bf27def29ef1ca7f5bfe20c1306bf78087ea29).